docs/weave-net-support.md

```diff
@@ -0,0 +1,67 @@
+# Setup WeaveNet CNI monitoring using kube-prometheus
+[WeaveNet](https://kubernetes.io/docs/concepts/cluster-administration/networking/#weave-net-from-weaveworks) is a resilient and simple to use CNI for Kubernetes. A well monitored and observed CNI helps in troubleshooting Kubernetes networking problems. [WeaveNet](https://www.weave.works/docs/net/latest/concepts/how-it-works/) emits [prometheus metrics](https://www.weave.works/docs/net/latest/tasks/manage/metrics/) for monitoring WeaveNet. There are many ways to install WeaveNet in your cluster. One of them is using [kops](https://github.com/kubernetes/kops/blob/master/docs/networking.md).
```

Comment:
> simple to use CNI

Do you mean "CNI plugin" or maybe "CNI provider"? CNI is an interface, not a concrete thing

Reply:
This is still outstanding, it doesn't really mean anything to be `a CNI`